shell/platform/linux/fl_renderer.h

```diff
@@ -78,53 +78,20 @@ struct _FlRendererClass {
                        gint scale);
 };
 
-/**
- * fl_renderer_setup:
- * @renderer: an #FlRenderer.
- * @error: (allow-none): #GError location to store the error occurring, or %NULL
- * to ignore.
- *
- * Set up the renderer.
- *
- * Returns: %TRUE if successfully setup.
- */
-gboolean fl_renderer_setup(FlRenderer* renderer, GError** error);
-
-/**
- * fl_renderer_get_visual:
- * @renderer: an #FlRenderer.
- * @screen: the screen being rendered on.
- * @error: (allow-none): #GError location to store the error occurring, or %NULL
- * to ignore.
- *
- * Gets the visual required to render on.
- *
- * Returns: a #GdkVisual.
- */
-GdkVisual* fl_renderer_get_visual(FlRenderer* renderer,
-                                  GdkScreen* screen,
-                                  GError** error);
-
-/**
- * fl_renderer_set_window:
- * @renderer: an #FlRenderer.
- * @window: the GDK Window this renderer will render to.
- *
- * Set the window this renderer will use.
- */
-void fl_renderer_set_window(FlRenderer* renderer, GdkWindow* window);
-
 /**
  * fl_renderer_start:
  * @renderer: an #FlRenderer.
+ * @widget: the application's main GTK widget.
```

Comment:
This is the widget Flutter is being rendered to, which may not be the main widget, right?